File: backend/src/main/java/com/NBE_4_5_2/Team5/domain/post/post/entity/ProductMetadata.java
package com.NBE_4_5_2.Team5.domain.post.post.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import lombok.Getter;

@Entity
@Getter
public class ProductMetadata {

	@Id
	private String name;
	@Column(name = "metadata_value")
	private String value;

	public void setValue(String value) {
		this.value = value;
	}

	public ProductMetadata(String name, String value) {
		this.name = name;
		this.value = value;
	}

	public ProductMetadata() {
	}
}

File: backend/src/main/java/com/NBE_4_5_2/Team5/domain/post/post/entity/ProductMetadata.kt
package com.NBE_4_5_2.Team5.domain.post.post.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id

@Entity
class ProductMetadata() {
    @Id
    lateinit var name: String

    @Column(name = "metadata_value")
    lateinit var value: String

    constructor(name: String, value: String): this() {
        this.name = name
        this.value = value
    }
}
